crates/libs/core/src/event.rs

```diff
@@ -16,6 +16,9 @@ pub struct Event<T: Interface> {
     delegates: UnsafeCell<Array<T>>,
 }
 
+unsafe impl<T: Interface> Send for Event<T> {}
+unsafe impl<T: Interface> Sync for Event<T> {}
```

Comment:
The `Send` and `Sync` implementations seem to have disappeared.

Reply:
Yup accidentally removed it, now it's back